packages/components/Tabs/src/Tabs.tsx

```diff
@@ -98,17 +98,25 @@ export const Tabs = compose<TabsType>({
 
     // Populate the tabsItemKeys array
     if (children) {
+      const enabledKeys = []
```

Comment:
I'm wondering if this whole `enabledKeys` logic is actually needed. The whole point of enabledKeys is to find the selectedKey, correct? 

Why not just use `data.selectedKey`?

Reply:
The point of enabledKeys is to find an available key to be selected as a new TabsItem if the currently selected has been hidden. For this scenario, I can't see how data.selectedKey would help find the other key that should be selected.